### DISCUSSION TITLE
Can someone explain what am I doing wrong?

### DISCUSSION THREAD

**Discussion by GeekyTechnician** [vosmiic/jellyfin-ani-sync#137] [Q&A] [answered]
I cannot get the test authentication to work. I thought I put everything right but, it keeps failing when testing authentication.
Jellyfin is in a docker container on Unraid, version 10.9.4.
Here's the log
https://pastebin.com/s2MP623B

**Comment by vosmiic** [comment, marked as answer]
Just to confirm, you have followed [these](https://github.com/vosmiic/jellyfin-ani-sync/wiki/Installation-&-Config#authorize) instructions right? Did you get the "Thank you for authenticating" message after authorizing?

If so, please confirm that an access token exists in your plugin config XML file (can be found [here](https://jellyfin.org/docs/general/server/plugins/)).

**Reply by GeekyTechnician** [reply]
No, that's it, somehow I skipped over the authentication portion completely. That's what I get for doing it in a hurry, thank you!

**Reply by vosmiic** [reply]
No worries

**Reply by GeekyTechnician** [reply]
> No worries

Hey, so, I never really managed to get it working, and I'm trying to again, this time I read the instructions carefully. But when copying the link to either anilist or MAL and then hitting the authorize/authenticate button I get an error page that says "This site can’t provide a secure connection" and the authentication won't go through. If I do the APIURL test I get the "this is the correct URL message" so I know that part must be correct. If it's relevant, everything is on local network, within Tailscale as VPN.

i believe this is the relevant data in the log

https://pastebin.com/KzScHP7B

**Reply by vosmiic** [reply]
So that confirms that the user is not authenticated with the API.
>But when copying the link to either anilist or MAL and then hitting the authorize/authenticate button I get an error page that says "This site can’t provide a secure connection" and the authentication won't go through

Just to confirm, the link you are visiting is the one that is generated by the plugin right? If you are attempting to authenticate with MAL the URL should look similar to https://myanimelist.net/v1/oauth2/authorize?response_type=code&client_id=***&code_challenge=***&redirect_uri=.../AniSync/authCallback

**Reply by GeekyTechnician** [reply]
Yeah, I get the correct page on either MAL or Anilist and I get the option to press the "authenticate" button, it just goes to that error page when I press it. So getting it authenticated is my issue. I tried doing it both on Firefox and Chrome and got an error. It happens with either MAL or Anilist. And yes, the link looks similar to what you posted.

**Reply by vosmiic** [reply]
Is the generated callback URL http or https? And do you connect to your Jellyfin instance through https?

**Reply by GeekyTechnician** [reply]
Generated callback is https.....Jellyfin runs on "non-S"
Is that my issue then?

**Reply by vosmiic** [reply]
If the test API URL that returns the "this is the correct URL" message is http, then probably. Try changing the callback URL to http and try again.
